go-controller/pkg/clustermanager/userdefinednetwork/controller_test.go

```diff
@@ -76,7 +77,7 @@ var _ = Describe("User Defined Network Controller", func() {
 			It("should create NAD successfully", func() {
 				udn := testUDN()
 				expectedNAD := testNAD()
-				c = newTestController(renderNadStub(expectedNAD), udn)
+				c = newTestController(renderNadStub(expectedNAD), udn, testNamespace("test"))
```

Comment:
Question: are these tests fail if we dont pass the test-namespace object?

Reply:
in almost all cases yes cause it needs to check the namespace, kidn of makes sense for the test with NAD to also include a namespace though